roles/python/tasks/main.yml

```diff
@@ -2,24 +2,11 @@
 
 - name: Install some apt packages
   apt:
-    name: '{{ item }}'
-  with_items:
-    - libyaml-dev
+    name: '{{ packages }}'
```

Comment:
Too much complexity here. Just list packages under name directly.

Reply:
Do you mean:
```
    name:
        - package1
        - package2
```

?

Reply:
Yep. I would understand if you wanted to put them into a dedicated place somewhere else. But declaring vars in a task to reuse them one line above is weird.

Reply:
I took it from the example in the Ansible documentation: https://docs.ansible.com/ansible/latest/modules/apt_module.html#examples

Reply:
But it looks better your way, I just wasn't sure what was the YAML syntax for that

Reply:
Oh, our docs probably need updating :)

In general, YAML knows JSON-like syntax for sequences `name: [package1, package2]` and human-readable:
```yaml
name:
- package1
- package2
```
I prefer to not do indentation because it already knows that it's a sequence from the `-` sign.

When you use `name: "{{ some_var }}"` Ansible renders it as a Jinja2 template and preserves type, so that it's actually a list, not a string :)